src/components/Input/Input.js

```diff
@@ -14,83 +14,81 @@ import {
   InputRequiredStyled
 } from './InputStyled';
 
-class Input extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    const {
-      type,
-      placeholder,
-      value,
-      onChange,
-      onBlur,
-      error,
-      showVisibilityIcon,
-      handleClickShowPassword,
-      showPassword,
-      passwordStrength,
-      ariaRequired,
-      ariaInvalid,
-      icon,
-      required,
-      floatingLabels,
-      reference
-    } = this.props;
-
-    return (
-      <InputComponentStyled>
-        <InputElementWrapperStyled
-          error={error}
-          passwordStrength={passwordStrength}
+const Input = ({
+  type,
+  placeholder,
+  value,
+  onChange,
+  onBlur,
+  error,
+  showVisibilityIcon,
+  handleClickShowPassword,
+  showPassword,
+  passwordStrength,
+  ariaRequired,
+  ariaInvalid,
+  icon,
+  required,
+  floatingLabels,
+  reference
```

Comment:
worth checking if we are using all of these props when embedding an Input component. Maybe we could remove unused ones

Reply:
It looks that all of the props are used but maybe in the process of refactoring components like DateInput, EmailInput some will be deleted. 
![image](https://user-images.githubusercontent.com/19149811/197751770-2f7e54ff-5c22-422d-8f69-e741d571b479.png)